src/store/migrations/update-default-name.js

```diff
@@ -0,0 +1,22 @@
+export default {
+  migrate(state, store) {
+    return new Promise(async resolve => {
```

Comment:
```suggestion
  async migrate(state, store) {
```
but actually, I think this migration should run in sync